src/xclim/indices/_conversion.py

```diff
@@ -1319,6 +1319,58 @@ def _get_D_from_M(time):  # noqa: N802
     )
 
 
+# TODO: How can I check exact units for inputs
+# @declare_units(
+#     net_radiation="[MJ m-2 day-1]",
+#     tas="[degC]",
+#     wind="[m s-1]",
+#     es="[kPa]",
+#     ea="[kPa]",
+#     delta_svp="[kPa degC-1]",
+#     gamma="[kPa degC]",
+#     G="[MJ m-2 day-1]",
+# )
```

Comment:
I think other converters don't use the declare_units. It might  trigger some failures in the code since you're using unreserved variable names for this. You can just add a bunch of conversions in the function that will guard rail against bad unphysical units, and put things how you want
```suggestion
```

Reply:
oups, just saw your comment @aulemahal , we're saying the same thing, but I was proposing to simply drop the declareunits like other converters

Reply:
Indeed, the  `declare_units` really is needed only if there's a plan to make this function an indicator itself.